src/components/Footer/styles.css

```diff
@@ -0,0 +1,521 @@
+@import url('https://rsms.me/inter/inter.css');
+html { font-family: 'Inter', sans-serif; }
+@supports (font-variation-settings: normal) {
+  html { font-family: 'Inter var', sans-serif; 
+    background: #F2F2F2;}
+}
+
+footer {
+    background: #FFFFFF;
+}
+
+#footer-component {
+    width: 100%;
+    display: grid;
+    justify-content: center;
+    text-align: center;
+    background: #FFFFFF;
+    margin: 0px auto 0px auto;
+    bottom:0; 
+}
+
+#become-nanny {
+    width: 473.8px;
+    height: 24px;
+    margin: 0 0 20px;
+    font-family: Inter;
+    font-size: 28px;
+    font-weight: 500;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 0.86;
+    letter-spacing: normal;
+    text-align: center;
+    color: #4a4a4a;
+}
+
+#become-nanny-h1 {
+    margin-top: 4rem;
+}
+
+#become-nanny-mobile {
+    display: none;
+}
+
+#become-nanny-h1-mobile {
+    display: none;
+}
+
+#footer-component p {
+    width: 362px;
+    height: 21px;
+    margin: 0px 65px 39px 65px;
+    font-family: Inter;
+    font-size: 19px;
+    font-weight: normal;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: normal;
+    text-align: center;
+    color: #4a4a4a;
+}
+
+ #footer-component button {
```

Comment:
Avoid selecting tags. A good selector here would be `footer__button`

I like to follow BEM naming convention when we don't have css modules (something you'll learn in the future)

It's a guideline for your class names. It stands for BLOCK__ELEMENT--MODIFIER.

Block = Major component you're working on, i.e. `footer`
Element = Element inside this component, i.e. `button`. `footer__button`.
Modifier = Modifier of your component in case there's one, i.e. `active`. `.footer__button--active`

It helps projects to keep organized and avoid style override